src/hexo/themes/documentation/source/images/results-error-icon.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13.19 11.23"><defs><style>.cls-1{fill:#b91b06;}</style></defs><title>error-badge</title><path class="cls-1" d="M7.53 5.61l4.13-4.13a.66.66 0 0 0-.93-.93L6.59 4.68 2.46.55a.66.66 0 0 0-.93.93l4.13 4.13-4.13 4.14a.66.66 0 0 0 0 .93.68.68 0 0 0 .93 0l4.13-4.13 4.13 4.13a.66.66 0 0 0 .93-.93z"/></svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13.19 11.23"><path d="M7.53 5.6l4.13-4.12a.66.66 0 0 0-.93-.93L6.6 4.68 2.45.55a.66.66 0 0 0-.93.93L5.66 5.6 1.53 9.76a.66.66 0 0 0 0 .93.68.68 0 0 0 .93 0L6.6 6.54l4.12 4.13a.66.66 0 0 0 .93-.93z" fill="#b91b06"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?